Source/include/AlignedAllocator.h

```diff
@@ -23,6 +25,8 @@ namespace AlignedStore
         return ptr;
 #elif (ANDROID)
         return memalign(alignment, size);
+#elif defined(__XTENSA__)

```

Comment:
Surprised to see XTENSA referenced here.

Reply:
We had agreement with the Xbox team to share XTENSA references with source code partners, but we should remove XTENSA references from public code